src/v2/api/controllers/UserController.ts

```diff
@@ -228,7 +236,24 @@ export class UserController {
   }
 
   @Access('users.$userId.update')
+  @ApiBearerAuth()
   @Patch('/:userId/avatar')
+  @ApiOkResponse({
+    type: UserResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+                  InvalidEntityIdException: User with such id is not found\n
+                  DataNotFoundException: Data was not found`,
+  })
+  @ApiUnsupportedMediaTypeResponse({
+    description: `InvalidExtensionException:\n
+                  File extension is wrong`,
+  })
+  @ApiPayloadTooLargeResponse({
+    description: `TooLargeSizeException:\n
+                  The file size exceeds 1.5 MB`,
+  })
   @UseInterceptors(FileInterceptor('avatar'))
   async uploadAvatar (
```

Comment:
You must describe uploaded file (avatar)
Example: https://stackoverflow.com/questions/66605192/file-uploading-along-with-other-data-in-swagger-nestjs